src/components/dialogs/network-modifications/line/creation/line-creation-dialog.js

```diff
@@ -349,13 +353,41 @@ const LineCreationDialog = ({
         [editData, studyUuid, currentNodeUuid, snackError, onCreateLine]
     );
 
-    const onValidationError = (errors) => {
+    //scroll to the first array error (limits) when submitting the form
+    useEffect(() => {
+        if (isSubmitting && errors && Object.keys(errors).length > 0) {
+            if (
+                tabIndexesWithError.includes(
+                    LineCreationDialogTab.LIMITS_TAB
+                ) &&
+                tabIndex === LineCreationDialogTab.LIMITS_TAB
+            ) {
+                let firstArrayWithErrors = Object.keys(errors[LIMITS])[0];
+                let temporaryLimitsErrors =
+                    errors[LIMITS][firstArrayWithErrors]?.[TEMPORARY_LIMITS];
+                if (temporaryLimitsErrors?.ref) {
+                    //case where ErrorInput is rendered
+                    let errorId = temporaryLimitsErrors.ref.name;
+                    let errorInput = document.getElementById(errorId);
+                    errorInput.scrollIntoView({
+                        behavior: 'smooth',
+                        block: 'nearest',
+                    });
+                }
+            }
+        }
+    }, [tabIndexesWithError, errors, isSubmitting, tabIndex]);
```

Comment:
I think it would be cleaner to do this in "onValidationError"

"errors" object is updated each time the user will update field contents, this useEffect could execute more often than expected

Reply:
i did it in useEffect because i want to do the scroll AFTER the tab is changed
and it does not trigger the useeffect everytime 'errors' is changed, i tested it